lib/data/entity.js

```diff
@@ -417,6 +475,26 @@ const getWithConflictDetails = (defs, audits, relevantToConflict) => {
 
   const relevantBaseVersions = new Set();
 
+  // build up branches
+  const branches = new Map();
+  for (const version of defs) {
+    const { branchId } = version;
+    if (branchId != null && version.branchBaseVersion != null) {
```

Comment:
I added `version.branchBaseVersion != null` to Frontend mostly as a temporary measure, since at the time, there were submissions where the entity creation got a branch ID. I think we can remove the condition here.